docker-compose-postgres-port.yml

```diff
@@ -0,0 +1,61 @@
+---
+services:
+  riven-frontend:
+    image: spoked/riven-frontend:latest
+    container_name: riven-frontend
+    restart: unless-stopped
+    ports:
+      - "3000:3000"
+    tty: true
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - ORIGIN=http://localhost:3000
+      - BACKEND_URL=http://127.0.0.1:8080
+      - TZ=Europe/Amsterdam
+    depends_on:
+      riven:
+        condition: service_healthy
+      
+  riven:
+    image: spoked/riven:latest
+    container_name: riven
+    restart: unless-stopped
+    ports:
+      - "8080:8080"
+    tty: true
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Amsterdam
+      - RIVEN_DATABASE_HOST=postgresql+psycopg2://postgres:postgres@riven_postgres/riven
```

Comment:
there doesn't need to be 3 composes for this. Just add a comment at the end of the line for `RIVEN_DATABASE_HOST` with the example of the sqlite database. Which should be set to `sqlite:////riven/data/media.db`

Reply:
Just added them for Mini since there wasnt any, just check if you see correct one which you like to use.